pdns/lwres.cc

```diff
@@ -64,6 +64,37 @@ std::shared_ptr<Logr::Logger> g_slogout;
 
 bool g_syslog;
 
+void remoteLoggerQueueData(RemoteLoggerInterface& r, const std::string& data)
+{
+  auto ret = r.queueData(data);
+
+  switch (ret) {
+  case RemoteLoggerInterface::Result::Queued:
+    break;
+  case RemoteLoggerInterface::Result::PipeFull: {
+    const auto name = r.name();
+    const auto msg = "queue full, dropping";
```

Comment:
Perhaps we could have a method returning a `const string&` from a `RemoteLoggerInterface::Result` instead of duplicating these messages (dnsdist and rec)?